eng/AzureTestLab.ps1

```diff
@@ -242,6 +242,7 @@ function Invoke-TestRunnerFromTeamCity($testType) {
         $zipReportPath
     }
 
+    $ErrorActionPreference = 'SilentlyContinue'
```

Comment:
May I suggest adding a comment about why you want to silently continue, thereby ignoring errors?